docker/index.yaml

```diff
@@ -24,7 +24,7 @@
 artifactory.algol60.net/sat-docker/stable:
   images:
     cray-sat:
-      - 3.25.0
+      - 3.25.5
```

Comment:
Oh, I'm going to bump this to `3.25.6`, which I just released. It fixes an unrelated bug (CRAYSAT-1780).